runtime/runtime-params-estimator/src/gas_cost.rs

```diff
@@ -153,6 +154,30 @@ impl GasCost {
             self.uncertain = rhs.uncertain;
         }
     }
+    /// JSON representation of the gas cost. This is intended to be used by
+    /// other scripts, such as the continuous estimation pipeline. Consumers
+    /// should expect more fields to be added. But existing fields should remain
+    /// stable.
+    pub fn to_json(&self) -> serde_json::Value {
+        match self.metric {
+            GasMetric::ICount => json!({
+                "gas": self.to_gas(),
+                "metric": "icount",
+                "instructions": self.instructions.to_f64(),
+                "io_r_bytes": self.io_r_bytes.to_f64(),
+                "io_w_bytes": self.io_w_bytes.to_f64(),
+                "uncertain": self.uncertain.is_some(),
+                "uncertain_reason": self.uncertain.map(|u| u.reason),
```

Comment:
feels like this can be just one field perhaps? No strong opinion though

Reply:
Yes that's actually nicer, it removes unnecessary redundancy from the output and streamlines how script should use it. I changed it so we only have `uncertain_reason` which is `null` if the result is not uncertain.